tensorflow/contrib/rnn/kernels/BUILD

```diff
@@ -0,0 +1,19 @@
+# Libraries and Kernels for RNNs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "lstm_ops",
+    prefix = "lstm",
+    deps = [
+        "//tensorflow/contrib/rnn:lstm_ops_op_lib",
+        "//tensorflow/core:framework",
```

Comment:
Internally `tf_custom_op_library` does turn in to separate targets for gpu files and other files.